contrib/initramfs/scripts/zfs

```diff
@@ -616,21 +616,21 @@ ask_user_snap()
 
 setup_snapshot_booting()
 {
-	snap="$1"
+	boot_snap="$1"
```

Comment:
Technically not part of the original problem, but if we're going to rename parameter variables, better do it everywhere just to be safe and be consistent.